content/tutorials/themes/theme-objects/index.md

```diff
@@ -113,13 +113,11 @@ Where the "Register" block should be on top of the ASCX file.
 The <dnn:BREADCRUMB...> block should be placed in the Theme on the spot you want it to appear.
 
 > [!NOTE]
-> The following list is far from complete (some work to be done) and some of the Theme Objects in the list are legacy
+> The following list are the Core Theme / Container Objects.
 
 |**Theme Object**|**Description**|
 |---|---|
-|ACTIONBUTTON|Displays an action from the **module action menu**.|
 |BREADCRUMB|Displays the path to the current tab (`>` is the default separator). Example: `PageName1 > PageName2 > PageName3`|
```

Comment:
Here is where you would link `BREADCRUMB` to the new page in #473

Reply:
@david-poindexter done at #473